engine/battle/init_battle_variables.asm

```diff
@@ -20,12 +20,13 @@ InitBattleVariables:
 	ld [hli], a ; wPlayerHPBarColor
 	ld [hl], a ; wEnemyHPBarColor
 	ld hl, wCanEvolveFlags
-	ld b, $3c
+	ld b, wMiscBattleDataEnd - wMiscBattleData
 .loop
 	ld [hli], a
 	dec b
 	jr nz, .loop
-	inc a ; POUND
+	; Force player to use POUND (move index 1) during TestBattle.
+	inc a
```

Comment:
This is redundant given the WRAM label name. Also `inc a ; SOME_CONSTANT_ONE` is common.
```suggestion
	inc a ; POUND
```